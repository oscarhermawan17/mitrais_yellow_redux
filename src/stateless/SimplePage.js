import React from 'react';
import PropTypes from 'prop-types';

const SimplePage = (props) =>
  <div className="simple_page oke">
      {props.title}
  </div>

SimplePage.propTypes = {
    title: PropTypes.string
};

export default SimplePage;