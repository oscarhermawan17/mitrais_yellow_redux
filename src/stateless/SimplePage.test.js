import { shallow } from 'enzyme'
import React from 'react'
import SimplePage from './SimplePage'

describe("SimplePage Stateless", () => {
    it("snapshot", () => {
        expect(shallow(<SimplePage />)).toMatchSnapshot()
    })
})