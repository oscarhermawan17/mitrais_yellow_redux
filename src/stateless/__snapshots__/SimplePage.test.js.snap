// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Stateless snapshot 1`] = `ShallowWrapper {}`;

exports[`SimplePage Stateless snapshot 1`] = `ShallowWrapper {}`;
